manifest/dev.baru/ResoniteScreenshotExtensions/info.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "ResoniteScreenshotExtensions",
+	"id": "dev.baru.resonite.ResoniteScreenshotExtensions",
+	"description": "Extensions of screenshot",
```

Comment:
could do with being more descriptive of atleast a feature of what this mod does

Reply:
I made the change
If you have a better explanation, please suggest it. I am not good at English as I use translations